src/other/acr/demo-wcag21/assessment-wcag21-SC1_1_1-a.json

```
@@ -0,0 +1,167 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": "earl:Assertion",
```

Add a reference to the pre-defined rule sets used for the production of this assessment

```suggestion
  "@type": "earl:Assertion",
  "dct:source": "act:rulesets/wcag2x/sc/1_1_1.json",
```

---

I added the dct:source